Neos.Flow/Classes/Aop/Builder/AdvicedConstructorInterceptorBuilder.php

```diff
@@ -40,6 +40,11 @@ public function build(string $methodName, array $interceptedMethods, string $tar
         $declaringClassName = $interceptedMethods[$methodName]['declaringClassName'];
         $proxyMethod = $this->compiler->getProxyClass($targetClassName)->getConstructor();
         if ($declaringClassName !== $targetClassName) {
+            /**
+             * @phpstan-ignore-next-line "buildMethodParametersCode" does not exist, and never has since the introduction of this call:
```

Comment:
Interesting… the method does exist, but not in a way that easily explains why it would be accessible on `$this` at that point. 🤔 Maybe @robertlemke remembers more?

Reply:
Well i just think this codepath is never taken ...

Reply:
This dates back to Andi and I believe the solution is as simple as this being 
`$proxyMethod->setMethodParametersCode($proxyMethod->buildMethodParametersCode($declaringClassName, $methodName, true));`

Which becomes even clearer looking at the sibling code in `AdvicedMethodInterceptorBuilder` where it is like I suggested

Reply:
but it seems no one ever noticed this, and it never caused trouble? So its a dead code path?